Program.cs

```diff
@@ -13,11 +13,113 @@ class Program
     {
         static void Main(string[] args)
         {
-            Parser.Default.ParseArguments<Options>(args)
-                .WithParsed<Options>(options => RunOptionsAndReturnExitCode(options))
-                .WithNotParsed<Options>((errors) => HandleParseErrors(errors));
+            var parser = new Parser(o =>
+            {
+                o.HelpWriter = null;
+            });
+
+            var result = parser.ParseArguments<Options, InitOptions, BumpMajorVersionOptions, BumpMinorVersionOptions>(args);
+
+            if ((result is NotParsed<object>) && ((NotParsed<object>)result).Errors.Any(o => o.Tag == ErrorType.NoVerbSelectedError || o.Tag == ErrorType.BadVerbSelectedError))
```

Comment:
`repo-version --help` no longer shows the options for the default verb, and there is currently no way to see them.

Reply:
parsing args natively now